exercises/robot-simulator/test/Tests.hs

```diff
@@ -1,48 +1,105 @@
-import Test.HUnit (Assertion, (@=?), runTestTT, Test(..), Counts(..))
-import System.Exit (ExitCode(..), exitWith)
-import Robot (Bearing(..), Robot, mkRobot,
-              coordinates, simulate,
-              bearing, turnRight, turnLeft)
+{-# OPTIONS_GHC -fno-warn-type-defaults #-}
 
-exitProperly :: IO Counts -> IO ()
-exitProperly m = do
-  counts <- m
-  exitWith $ if failures counts /= 0 || errors counts /= 0 then ExitFailure 1 else ExitSuccess
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
 
-testCase :: String -> Assertion -> Test
-testCase label assertion = TestLabel label (TestCase assertion)
+import Robot
+  ( Bearing ( East
+            , North
+            , South
+            , West
+            )
+  , bearing
+  , coordinates
+  , mkRobot
+  , simulate
+  , turnLeft
+  , turnRight
+  )
 
 main :: IO ()
-main = exitProperly $ runTestTT $ TestList
-  [ TestList robotTests ]
-
-robotTests :: [Test]
-robotTests =
-  [ testCase "turning cases" $ do
-    West  @=? turnLeft North
-    South @=? turnLeft West
-    East  @=? turnLeft South
-    North @=? turnLeft East
-    East  @=? turnRight North
-    South @=? turnRight East
-    West  @=? turnRight South
-    North @=? turnRight West
-  , testCase "robbie" $ do
-    let robbie :: Robot
```

Comment:
guess it's always been that way (assume we ported from clojure) https://github.com/exercism/exercism.io/commit/33216cfbcd78086400a7f03167ae9dba497afded